Classes/Command/NodeCommandController.php

```diff
@@ -14,7 +14,7 @@
 use Doctrine\ORM\EntityManager;
 use Doctrine\ORM\Query;
 use Neos\Flow\Annotations as Flow;
-use Neos\Flow\Object\ObjectManagerInterface;
+use Neos\Flow\ObjectManagement\ObjectManagerInterface;
```

Comment:
This is a quote serious error and not a "minor phpstorm error"? I mean, just using the old, now obsolete and non existent path will lead to a fatal error.